.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build
+
+on:
+    push:
+        branches: [ v1 ]
+    pull_request:
+        branches: [ v1 ]
+
+jobs:
+    build:
+        name: Run Typescript Build
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 16
```

Comment:
Is this our standard version of node? I thought it was 18?

Reply:
It seems we are currently using types for node 16 which is why I am setting it as such. But I could upgrade to 18 possibly for this.